File: Code coach problems/Easy/C++/Popsicles.cpp
#include <iostream>
using namespace std;

int main() {

	int a;
	int b;
	
	cin>>a>>b;
	
	if (b>=a && b%a == 0)
		cout << "give away";
	else
		cout << "eat them yourself";
		
	return 0;
}

File: Code coach problems/Easy/Java/Popsicles.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();

        if(b>=a && b%a == 0)
        {
            System.out.println("give away");
        }
        else
        {
            System.out.println("eat them yourself");
        }

       
    }
}
